lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -44,40 +46,46 @@ class Meta:
         ]
 
 
-class ScoreSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+class InitializeSerializer(serializers.Serializer):
     """
-    Score (points earned/possible) for use in SubmissionMetadataSerializer
+    Serialize info for the initialize call. Packages ORA, course, submission, and rubric data.
     """
-    pointsEarned = serializers.IntegerField()
-    pointsPossible = serializers.IntegerField()
+    courseMetadata = CourseMetadataSerializer()
+    oraMetadata = OpenResponseMetadataSerializer()
+    submissions = serializers.DictField()
```

Comment:
Ooh, how about this:
https://www.django-rest-framework.org/api-guide/fields/#dictfield
> child - A field instance that should be used for validating the values in the dictionary. If this argument is not provided then values in the mapping will not be validated.

so     `submissions = serializers.DictField(child=SubmissionMetadataSerializer)`

Reply:
Then in the SubmissionMetadataSerializer,

```
    username = serializers.CharField(default=None)
    teamName = serializers.CharField(default=None)
```

The Score is still annoying there though....

Reply:
There's a complication in the structure of the submissions, where submissions are not a list but values of the key = submission UUID. This made using the serializers tricky.